glean-core/python/glean/metrics/event.py

```diff
@@ -125,7 +124,7 @@ def record(self, extra: Optional[Dict[int, str]] = None) -> None:
         if self._disabled:
             return
 
-        timestamp = _util.time_ms()
+        timestamp = _ffi.lib.glean_get_timestamp_ms()
```

Comment:
Maybe we should also remove `_util.py::time_ms` since this is the only use of it in the code base?